--- a/.config/constraints.txt
+++ b/.config/constraints.txt
@@ -41,7 +41,6 @@
 hjson==3.1.0              # via mkdocs-macros-plugin, super-collections
 htmlmin2==0.1.13          # via mkdocs-minify-plugin
 idna==3.10                # via requests
-importlib-metadata==8.6.1  # via ansible-lint (pyproject.toml)
 iniconfig==2.1.0          # via pytest
 ipdb==0.13.13             # via ansible-lint (pyproject.toml)
 ipython==8.34.0           # via ipdb, ansible-lint (pyproject.toml)
--- a/.config/requirements-lock.txt
+++ b/.config/requirements-lock.txt
@@ -8,7 +8,6 @@
 click==8.1.8              # via black
 cryptography==44.0.2      # via ansible-core
 filelock==3.18.0          # via ansible-lint (pyproject.toml)
-importlib-metadata==8.6.1  # via ansible-lint (pyproject.toml)
 jinja2==3.1.6             # via ansible-core
 jsonschema==4.23.0        # via ansible-compat, ansible-lint (pyproject.toml)
 jsonschema-specifications==2024.10.1  # via jsonschema
--- a/.config/requirements.in
+++ b/.config/requirements.in
@@ -4,7 +4,6 @@
 # alphabetically sorted:
 black>=24.3.0 # MIT (security)
 filelock>=3.8.2 # The Unlicense
-importlib-metadata # Apache
 jsonschema>=4.10.0 # MIT, version needed for improved errors
 packaging>=22.0 # Apache-2.0,BSD-2-Clause
 pathspec>=0.10.3 # Mozilla Public License 2.0 (MPL 2.0)
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -165,7 +165,6 @@
           - black>=22.10.0
           - cryptography>=39.0.1
           - filelock>=3.12.2
-          - importlib_metadata
           - jinja2
           - license-expression >= 30.3.0
           - pip>=22.3.1
--- a/src/ansiblelint/requirements.py
+++ b/src/ansiblelint/requirements.py
@@ -1,6 +1,6 @@
 """Utilities for checking python packages requirements."""
 
-import importlib_metadata
+import importlib.metadata
 from packaging.requirements import Requirement
 from packaging.specifiers import SpecifierSet
 from packaging.version import Version
@@ -13,7 +13,7 @@
 
     def __init__(self, name: str = "ansible-lint") -> None:
         """Load linter metadata requirements."""
-        for req_str in importlib_metadata.metadata(name).json["requires_dist"]:
+        for req_str in importlib.metadata.metadata(name).json["requires_dist"]:
             req = Requirement(req_str)
             if req.name:
                 self[req.name] = req.specifier
